eng/WpfArcadeSdk/tools/ReferenceAssembly.targets

```diff
@@ -1,5 +1,59 @@
 <Project>
-  
+
+  <PropertyGroup>
+    <_DotnetApiDocsFilesRoot>$([MSBuild]::NormalizeDirectory('$(NuGetPackageRoot)', 'microsoft.private.intellisense', '$(MicrosoftPrivateIntellisenseVersion)', 'IntellisenseFiles', 'net'))</_DotnetApiDocsFilesRoot>
```

Comment:
Have you tested this? I don't think this is the correct path in the new intellisense package.